djmoney/forms/fields.py

```diff
@@ -17,17 +18,34 @@ class MoneyField(MultiValueField):
     if not hasattr(MultiValueField, 'empty_values'):
         empty_values = list(validators.EMPTY_VALUES)
 
-    def __init__(self, currency_widget=None, currency_choices=CURRENCY_CHOICES, choices=CURRENCY_CHOICES,
-                 max_value=None, min_value=None,
+    def __init__(self, currency_widget=None, currency_choices=CURRENCY_CHOICES,
+                 choices=CURRENCY_CHOICES, max_value=None, min_value=None,
                  max_digits=None, decimal_places=None, *args, **kwargs):
+
+        # choices does not make sense in this context, it would mean we had
+        # to replace two widgets with one widget dynamically... which is a
+        # mess. Instead, we let currency_choices be the same as choices and
+        # raise a warning.
         if currency_choices != CURRENCY_CHOICES:
             warn('currency_choices will be deprecated in favor of choices', PendingDeprecationWarning)
             choices = currency_choices
-        decimal_field = DecimalField(max_value, min_value, max_digits, decimal_places, *args, **kwargs)
-        choice_field = ChoiceField(choices=currency_choices)
-        self.widget = currency_widget if currency_widget else MoneyWidget(amount_widget=decimal_field.widget,
-                                                                          currency_widget=choice_field.widget)
-        fields = (decimal_field, choice_field)
+
+        amount_field = DecimalField(max_value, min_value, max_digits, decimal_places, *args, **kwargs)
+        currency_field = ChoiceField(choices=choices)
```

Comment:
This is where the fix happened, before it was still doing `choices=currency_choices`